new.js

```diff
@@ -0,0 +1,3 @@
+// is_null your comment
+// Some average code

```

Comment:
```diff
- // Some average code
+ // Some {{ .funcComment }}
```

Change `{{ .funcComment }}` by commenting: `@CodeLingoBot rewrite set funcComment "your comment"`

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output : exec: "node": executable file not found in $PATH

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output : exec: "node": executable file not found in $PATH

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output : exec: "node": executable file not found in $PATH

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output /node_modules/parse-diff/index.js:120
    ref = current.changes, [recentChange] = slice.call(ref, -1);
                           ^

ReferenceError: Invalid left-hand side in assignment
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/diff.js:30:19)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output module.js:328
    throw err;
    ^

Error: Cannot find module 'parse-diff'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/diff.js:30:19)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output /diff.js:45
    file.chunks.forEach(
               ^

TypeError: Cannot read property 'forEach' of undefined
    at /diff.js:45:16
    at Array.forEach (native)
    at Object.<anonymous> (/diff.js:41:13)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
    at startup (node.js:136:18)
    at node.js:966:3
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output module.js:549
    throw err;
    ^

Error: Cannot find module '/home/dan/go/src/github.com/codelingo/platform/flow/bots/service/github/diff.js'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output /home/dan/go/src/github.com/codelingo/platform/flow/bots/service/github/poll/diff.js:45
    file.chunks.forEach(
                ^

TypeError: Cannot read property 'forEach' of undefined
    at /home/dan/go/src/github.com/codelingo/platform/flow/bots/service/github/poll/diff.js:45:17
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/home/dan/go/src/github.com/codelingo/platform/flow/bots/service/github/poll/diff.js:41:13)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output /home/dan/go/src/github.com/codelingo/platform/flow/bots/service/github/poll/diff.js:45
    file.chunks.forEach(
                ^

TypeError: Cannot read property 'forEach' of undefined
    at /home/dan/go/src/github.com/codelingo/platform/flow/bots/service/github/poll/diff.js:45:17
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/home/dan/go/src/github.com/codelingo/platform/flow/bots/service/github/poll/diff.js:41:13)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output module.js:328
    throw err;
    ^

Error: Cannot find module 'parse-diff'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/diff.js:30:19)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output module.js:328
    throw err;
    ^

Error: Cannot find module '/poll/diff.js'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Function.Module.runMain (module.js:442:10)
    at startup (node.js:136:18)
    at node.js:966:3
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output module.js:328
    throw err;
    ^

Error: Cannot find module '/poll/diff.js'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Function.Module.runMain (module.js:442:10)
    at startup (node.js:136:18)
    at node.js:966:3
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output /node_modules/parse-diff/index.js:120
    ref = current.changes, [recentChange] = slice.call(ref, -1);
                           ^

ReferenceError: Invalid left-hand side in assignment
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/diff.js:30:19)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output /node_modules/parse-diff/index.js:120
    ref = current.changes, [recentChange] = slice.call(ref, -1);
                           ^

ReferenceError: Invalid left-hand side in assignment
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/diff.js:30:19)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output /diff.js:45
    file.chunks.forEach(
               ^

TypeError: Cannot read property 'forEach' of undefined
    at /diff.js:45:16
    at Array.forEach (native)
    at Object.<anonymous> (/diff.js:41:13)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
    at startup (node.js:136:18)
    at node.js:966:3
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output /node_modules/parse-diff/index.js:120
    ref = current.changes, [recentChange] = slice.call(ref, -1);
                           ^

ReferenceError: Invalid left-hand side in assignment
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/diff.js:30:19)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output /node_modules/parse-diff/index.js:120
    ref = current.changes, [recentChange] = slice.call(ref, -1);
                           ^

ReferenceError: Invalid left-hand side in assignment
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/diff.js:30:19)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output : exec: "node": executable file not found in $PATH

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Node.js command failed with output /node_modules/parse-diff/index.js:120
    ref = current.changes, [recentChange] = slice.call(ref, -1);
                           ^

ReferenceError: Invalid left-hand side in assignment
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/diff.js:30:19)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
: exit status 1

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "brilliant code"

Reply:
Success! Committed rewritten file.